### PR TITLE
Weather App project

### PR DESCRIPTION
**Pull request by SofiBretz**
**In this project I:**

- Add the 'Weather API' and configure it with the logic of the app.
- Add the 'Webpack' and 'eslint' packages.
- Add style to the app.
- Add the readme file with the image example and the 'demo' link.
- Program the logic of the app.
- Separate the main module form the DOM.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+
+### Prerequisites
+
+- You have to get a text editor to inspect the code in your computer.
+
+### Install
+
+- Install a text editor (if you don't have already).
+- Clone the project to your computer.
+
```
Comment (shloch) [inline README.md:36]: specify the commands to install and setup the project

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```
Comment (shloch) [inline .eslintrc.json:19]: use the same linting config file provided by microverse ([ click here to access file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) )

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      weather.temperature.value = Math.floor(data.main.temp - KELVIN);
+      weather.description = data.weather[0].description;
+      weather.iconId = data.weather[0].icon;
+      weather.city = data.name;
+      weather.country = data.sys.country;
+    })
+    .then(() => {
+      displayWeatherInfo(iconElement, tempElement, descElement, locationElement, weather);
+    });
+}
```
Comment (shloch) [inline src/index.js:78]: ### Great work with the API...this is great

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  const { latitude } = position.coords;
+  const { longitude } = position.coords;
+  getWeatherAuto(latitude, longitude);
+}
+
+function showError(error) {
+  notificationElement.style.display = 'block';
+  notificationElement.innerHTML = `<p> ${error.message} </p>`;
+}
+
```
Comment (shloch) [inline src/index.js:22]: Look for a way to separate LOGICAL code from DOM. This file manipulates the API and DOM elements. You could create a file for Weather Api that does calls and pass data to another DOM file, where you do all the displays [ [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) ]

File: README.md (hunk 1 above)
Comment (SofiBretz) [reply README.md]: Thanks for this.

File: .eslintrc.json (hunk 2 above)
Comment (SofiBretz) [reply .eslintrc.json]: Done, thanks!

File: src/index.js (hunk 3 above)
Comment (SofiBretz) [reply src/index.js:78]: Thank you!

**Comment by SofiBretz** [comment]
**Summary of Changes:**

- Added the 'Microverse eslint' configuration version.
- Added the install commands on the readme file.
- Created a new function to handle DOM for the error and for the temperature display.
- ESlint fixed, no offenses detected.

Ready for a code review, thanks! :)

File: src/index.js (hunk 4 above)
Comment (SofiBretz) [reply src/index.js]: Done, thank you.

**Pull request merged**